baseframe/__init__.py

```diff
@@ -145,6 +151,20 @@ def init_app(self, app, requires=[], ext_requires=[], bundle_js=None, bundle_css
         networkbar_cache.init_app(app, config=nwcacheconfig)
         asset_cache.init_app(app, config=acacheconfig)
         cache.init_app(app)
+
+        required_dogpile_attrs = ['DOGPILE_CACHE_BACKEND', 'DOGPILE_CACHE_URLS', 'DOGPILE_CACHE_REGIONS']
+        for dogpile_attr in required_dogpile_attrs:
```

Comment:
Now this could simply be:

```python
for key, value in required_dogpile_attrs.items():  # Assuming this is the name of the dictionary.
    app.config.setdefault(key, value)
```